packages/interpretations/src/components/interpretations/InterpretationDialog.js

```diff
@@ -6,56 +6,50 @@ import defer from 'lodash/fp/defer';
 import i18n from '@dhis2/d2-i18n'
 import { compact } from 'lodash/fp';
 import SharingDialog from '@dhis2/d2-ui-sharing-dialog';
-import RichEditor from '../html-editor/RichEditor';
+import TextField from 'material-ui/TextField';
 
 const styles = {
     dialog: {
         maxWidth: 600,
         height: 500,
     },
+    textfield: {
+        width: '100%',
+    },
 };
 
 class InterpretationDialog extends Component {
     constructor(props) {
         super(props);
         this.state = {
             value: props.interpretation.text,
-            showEditor: true,
             sharingDialogIsOpen: false,
         };
-        this.save = this.hideEditorAndThen(this._save.bind(this));
-        this.cancel = this.hideEditorAndThen(this._cancel.bind(this));
-    }
-
-    hideEditorAndThen(fn) {
-        // Method componentWillUnmount of child components of Dialog are called *after* their nodes
-        // are removed from the DOM (bug in mui?), which triggers errors in CKEditor.destroy.
-        // Workaround: Manually unmount the component using a flag in state.
-        return (...args) =>
-            this.setState({ showEditor: false }, () => defer(() => fn(...args)));
+        this.save = this.save.bind(this);
```

Comment:
Same as in the CommentTextarea. Change functions to class properties to avoid using constructor.